bin/jfr

```diff
@@ -0,0 +1 @@
+.openjdk@11.pkg
```

Comment:
You should almost never use channels, instead use a specific version eg. `hermit install openjdk-11.0.10_9`